packages/web-components/fast-element/src/styles.ts

```diff
@@ -279,6 +303,13 @@ export function css(
             }
 
             styles.push(value);
+        } else if (value instanceof CSSDirective) {
```

Comment:
I had an idea when looking at the chain of conditionals. What if we put this check first? Then, we could allow the return result of `createCSS()` to actually be `ComposableStyles`. So we'd basically take the output of `value.createCSS()` and assign it to `value` so that the checks below would then handle the other types. This could let a `CSSDirective` return not only strings but also `ElementStyles` and `CSSStyleSheet` instances. Thoughts?

Reply:
Yea good call, I'll update